.github/workflows/tests-nightly.yml

```diff
@@ -167,7 +167,7 @@ jobs:
 
       - name: Cypress run electron
         id: cypress-electron
-        if: ${{ github.event.inputs.tests != 'Critical' && github.event_name != 'repository_dispatch' && contains(fromJSON('["chrome", "all"]'), steps.get-browsers.outputs.result) && ! cancelled() }}
+        if: ${{ github.event.inputs.tests != 'Critical' && github.event_name != 'repository_dispatch' && ! cancelled() }}
```

Comment:
Won't be better if we just change contains(fromJSON('["chrome", "all"]') to contains(fromJSON('["electron", "all"]')

Reply:
If we leave it like this, when you manually trigger tests for eg. on firefox, tests on electron and on firefox will be executed

Reply:
seems like there is input on top, that allows you to use the only browsers listed there, so there is no need to have additional validation here